presto-tests/src/test/java/io/prestosql/memory/TestMemoryManager.java

```diff
@@ -350,6 +351,19 @@ public void testQueryUserMemoryLimit()
                 .build();
         try (QueryRunner queryRunner = createQueryRunner(SESSION, properties)) {
             queryRunner.execute(SESSION, "SELECT COUNT(*), repeat(orderstatus, 1000) FROM orders GROUP BY 2");
+
+            Session session = testSessionBuilder()
```

Comment:
this seems redundant to `testSessionQueryMemoryPerNodeLimit` test, isn't it?

Reply:
yes, this is redundant, removing it.